perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_attribute_def_def_o365EmailAddresses_muTest.java

```diff
@@ -139,8 +140,8 @@ public void testCheckMissingUco() throws Exception {
 		attributeToCheck.setValue(Lists.newArrayList("my@example.com", "aaa@bbb.com"));
 		try {
 			classInstance.checkAttributeSemantics(session, member, attributeToCheck);
-			fail("should throw WrongAttributeValueException");
-		} catch (WrongAttributeValueException ex) {
+			fail("should throw WrongReferenceAttributeValueException");
```

Comment:
You forgot to change this fail message in other test methods in this class.

Reply:
The messages should be correct now.